src/dotnet-serve/Startup.cs

```diff
@@ -62,6 +63,12 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app)
         {
+            app.UseCors(corsPolicy =>
```

Comment:
This makes the server much more permissive by default. Maybe this should be opt-in. What do you think about `dotnet serve --cors`?

Reply:
I quickly checked nginx's default configuration, and it does not allow OPTIONS out of the box. It makes sense to make it opt-in by using `--cors` option.
And I would suggest that this option will by default allow all origin and methods. For configuring selected allowed origin and methods, we can use config file when issue [#10](https://github.com/natemcmaster/dotnet-serve/issues/10) is implemented.
Is that ok?

Reply:
Sounds good to me!

Reply:
Oh, and thanks for the pull request 😄 I appreciate your willingness to contribute. Other than this one usage question, it looks great. You've done a good job!

Reply:
Welcome, and thanks 😉. I will update the code based on the suggestion.